### PR TITLE
bug/384_flume_conf_file

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue https://github.com/telefonicaid/fiware-connectors/issues/348
* Unit tests not required
* (unofficial) e2e tests:

```
$ sudo rpm -i neore/rpm/RPMS/x86_64/cygnus-0.7.1-%\{_product_release\}.x86_64.rpm 
[INFO] Creating cygnus user
[INFO] Creating log directory
Done
$ ls /usr/cygnus/conf/
agent.conf.template  cygnus_instance.conf.template  krb5.conf.template  log4j.properties           matching_table.conf.template
agent_test.conf      flume-env.sh.template          krb5_login.conf     log4j.properties.template  README.md
```
* Assignee @vgarciag but LGTM from @fgalan and @gtorodelvalle is also required

### CODE REVIEW

File: flume/conf/README.md [hunk 1]
```diff
@@ -4,26 +4,26 @@ From version 0.6.0 Cygnus is able to start multiple instances by renaming and co
...
 ```
 
 Then each file can be edited in order to change the configuration that will affect to all intances of Cygnus.
 
-## 2- Files for each instance
+## Files for each instance
 
-The other files are `cygnus.conf.template` and `flume.conf.template`. These files should be copied 
-for each intantance of Cygnus that want to run.
+The other files are `cygnus.conf.template` and `flume.conf.template`. These files should be copied for each intantance of Cygnus that want to run.
```
Comment (fgalan) [inline flume/conf/README.md]: intantance  -> instance

File: flume/neore/scripts/package.sh [hunk 2]
```diff
@@ -70,6 +70,7 @@ function download_flume(){
 	_log "#### Cleaning the temporal folders... ####"
 	rm -rf ${RPM_SOURCE_DIR}/${FLUME_WO_TAR}
 	rm -rf ${FLUME_WO_TAR}/docs # erase flume documentation
+        rm ${FLUME_WO_TAR}/conf/flume-conf.properties.template # we will add our own templates
```
Comment (fgalan) [inline flume/neore/scripts/package.sh]: Github web diff shows indentation problem here. Maybe some tab/whitepace issue here?

File: flume/neore/scripts/package.sh [hunk 3]
```diff
@@ -98,14 +99,7 @@ function copy_cygnus_conf() {
...
 	mkdir -p ${RPM_SOURCE_DIR}/config
-	for file in $(ls ${BASE_DIR}/conf)
-	do
-		# file=$(basename ${file})
-		# renamed_file=${file%.template}
-		# cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/${renamed_file}
-
-		cp ${BASE_DIR}/conf/${file} ${RPM_SOURCE_DIR}/config/
-	done
+        cp ${BASE_DIR}/conf/* ${RPM_SOURCE_DIR}/config/
```
Comment (fgalan) [inline flume/neore/scripts/package.sh]: This is the command that "add[s] our own templates" ?

Comment (fgalan) [inline flume/neore/scripts/package.sh]: In addition, github diff shows one whitespace offset in the indentation also here...

Comment (frbattid) [inline flume/neore/scripts/package.sh]: What was done here was to remove unnecesary loop and leftover comments.

Regarding the indentantion issue, uff, my IDE shows everything OK... The problem is the orginal file was indented by using a single "big" tab that is equivalent to 2 tabs in my IDE (Netbeans).

File: flume/neore/scripts/package.sh (hunk 2 above)
Comment (frbattid) [inline flume/neore/scripts/package.sh]: Uff, my IDE shows everything OK... The problem is the orginal file was indented by using a single "big" tab that is equivalent to 2 tabs in my IDE (Netbeans).

File: flume/conf/README.md (hunk 1 above)
Comment (frbattid) [inline flume/conf/README.md]: Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/f5e6a3201684bd03c9276ef30df68b6a4f89079b

File: flume/neore/scripts/package.sh (hunk 2 above)
Comment (frbattid) [inline flume/neore/scripts/package.sh]: Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/f5e6a3201684bd03c9276ef30df68b6a4f89079b

File: flume/neore/scripts/package.sh (hunk 3 above)
Comment (frbattid) [inline flume/neore/scripts/package.sh]: Fixed in commit https://github.com/telefonicaid/fiware-connectors/commit/f5e6a3201684bd03c9276ef30df68b6a4f89079b

File: flume/neore/scripts/package.sh (hunk 2 above)
Comment (fgalan) [inline flume/neore/scripts/package.sh]: When you cannot trust in your IDE, then double checking this with an independent editing tool (such vi) helps ;)

**Comment by fgalan** [comment]
LGTM (passing the ball to others reviewers)

**Comment by vgarciag** [comment]
LGTM

**Comment by gtorodelvalle** [comment]
LGTM, although I have not been able to run the procedure since I have not the environment properly set yet :)

**Pull request merged**